.env.example

```diff
@@ -24,3 +24,5 @@ SURVEY_SERVICE_DB_DIALECT=postgres
 SURVEY_SERVICE_PG_POOL_MAX=5
 SURVEY_SERVICE_PG_POOL_MIN=0
 SURVEY_SERVICE_PG_POOL_IDLE=10000
+SURVEY_SERVICE_PG_SSL=false
```

Comment:
The other services' envvars for this look like `SURVEY_SERVICE_PG_SSL_ENABLED`, I'd prefer if we kept them consistent

Reply:
I agree. Unfortunately though, this variable already exists. I'd rather not change it because that would require changing all deployments where this service is used.